nixos/modules/programs/bash/bash.nix

```
@@ -123,20 +129,24 @@ in
         type = types.lines;
       };
 
-      enableCompletion = mkOption {
-        default = true;
-        description = ''
-          Enable Bash completion for all interactive bash shells.
-        '';
-        type = types.bool;
-      };
+      enableCompletion = mkEnableOption "Bash completion for all interactive bash shells";
```

This was default `true`, so you are going to have to something like:

```suggestion
      enableCompletion = mkEnableOption "Bash completion for all interactive bash shells" // {
        default = true;
      };
```

to preserve backwards compatibility

---

Oh, oops, thanks!

---

@MetaDark no worries. Thanks for contributing!